lib/mayaUsd/ufe/UsdContextOps.cpp

```diff
@@ -275,7 +287,11 @@ Ufe::ContextOps::Items UsdContextOps::getItems(
         int hasLayerEditorCmd{0};
         MGlobal::executeCommand("runTimeCommand -exists UsdLayerEditor", hasLayerEditorCmd);
         if (hasLayerEditorCmd) {
+#if UFE_PREVIEW_VERSION_NUM >= 2023
+            items.emplace_back(kUSDLayerEditorItem, kUSDLayerEditorLabel, kUSDLayerEditorImage);
```

Comment:
Added icon for the "USD Layer Editor" context menu item. And for the "Add New Prim" types below.